californium-core/src/main/java/org/eclipse/californium/core/network/Exchange.java

```diff
@@ -466,8 +489,9 @@ public boolean isComplete() {
 	public void setComplete() {
 		this.complete = true;
 		ExchangeObserver obs = this.observer;
-		if (obs != null)
+		if (null != obs) {
```

Comment:
@boaks, you seem to prefer this pattern of putting the *constant* to compare with in the first place. Personally, I find this much harder to read. So, unless you have a very strong opinion on this and I'd rather have you not change these comparisons but instead stick to the approach used throughout the rest of the code. Otherwise we should discuss on the mailing list and have a vote on whether we want to adopt this pattern in general.